roscala/src/main/scala/coop/rchain/rosette/Ob.scala

```diff
@@ -98,6 +95,8 @@ trait Ob extends Base {
                value: Int): Ob =
     ??? //TODO
 
+  // TODO: Revisit
```

Comment:
I recommend removing cruft like this, since if you need the old code, git has a copy; but I think it likely that you'll get back to it in the next PR, so I won't insist on it.

Reply:
Done